### ISSUE TITLE
error: TS2345

### ISSUE THREAD

**Issue by schildbach** [hsjoberg/fork-explorer#48]
I'm not sure what has been changed since I last successfully started fork-explorer, but right now I'm always getting this error when starting the server:

```
fork-explorer_1     | error: TS2345 [ERROR]: Argument of type 'Middleware<Record<string, any>, Context<Record<string, any>, Record<string, any>>>' is not assignable to parameter of type 'Middleware<Record<string, any>, Context<Record<string, any>>>'.
fork-explorer_1     |   Types of parameters 'context' and 'context' are incompatible.
fork-explorer_1     |     Type 'Context<Record<string, any>>' is not assignable to type 'Context<Record<string, any>, Record<string, any>>'.
fork-explorer_1     |       Types of property 'app' are incompatible.
fork-explorer_1     |         Type 'Application<State>' is not assignable to type 'Application<Record<string, any>>'.
fork-explorer_1     |           Property '#middleware' in type 'Application' refers to a different member that cannot be accessed from within type 'Application'.
fork-explorer_1     | app.use(router.routes());
fork-explorer_1     |         ~~~~~~~~~~~~~~~
fork-explorer_1     |     at file:///fork-explorer/index.ts:13:9
```

I noticed you pushed 8782314bfd76f6287393517664bbb598ce45ee3f which seems to be related, but rebasing my branch on current master didn't help.

**Comment by hsjoberg** [comment]
@schildbach I can reproduce the problem as we'll. I'll take a look.

**Comment by hsjoberg** [comment]
Fixed in 41ddfe5.

**Comment by schildbach** [comment]
Thanks, that fixed it.
